### PR TITLE
Added installation of Ruby and Fastlane in the Dockerfile

### PR DESCRIPTION
**Pull request by sharath-cliqz**
This Dockerfile will also be used for Continuous Deployment using Fastlane.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,16 +2,60 @@ FROM ubuntu:16.04
...
+        openjdk-8-jdk \
+        python-dev \
+        python-pip \
+        python-setuptools \
+        unzip \
+        uuid \
+        wget \
+        xz-utils \
+        zip && \
+    apt-get install -y \
```
Comment (chrmod) [inline Dockerfile]: those two can be merged :-)

File: Dockerfile [hunk 2]
```diff
@@ -44,11 +88,20 @@ RUN mkdir -p $ANDROID_HOME; \
 # Install Node.JS
 RUN cd /home/jenkins && \
     wget --output-document=nodejs.tgz --quiet \
-      'https://nodejs.org/dist/v8.9.3/node-v8.9.3-linux-x64.tar.xz' && \
+      'https://repository.cliqz.com/dist/android/nodejs/node-v8.9.3-linux-x64.tar.xz' && \
```
Comment (chrmod) [inline Dockerfile]: why not to use nvm as you are using rvm already?

File: Dockerfile [hunk 3]
```diff
@@ -44,11 +88,20 @@ RUN mkdir -p $ANDROID_HOME; \
...
-RUN npm install -g wd
\ No newline at end of file
+#Installation of 'yarn'; 'appium' & 'wd' for Integration Tests
+RUN npm install --global \
+    yarn \
+    appium \
+    wd
+
+#Install Ruby and Fastlane
+SHELL ["/bin/bash", "-l", "-c"]
```
Comment (chrmod) [inline Dockerfile]: this one should probably go up, near ENV

Comment (sharath-cliqz) [reply Dockerfile]: Makes sense. Will do it. 👍

File: Dockerfile (hunk 2 above)
Comment (sharath-cliqz) [reply Dockerfile]: True. Now using 'nvm'.

File: Dockerfile (hunk 1 above)
Comment (sharath-cliqz) [reply Dockerfile]: I actually wanted to keep it separate to make it easier to track. The 1st set are the dependencies for FF and the next is our own. Mixing them both would make it slightly difficult to track later on.

File: Dockerfile [hunk 4]
```diff
@@ -24,6 +68,19 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
...
+
+# Install nvm with node and npm
+RUN curl https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash && \
+    . $NVM_DIR/nvm.sh && \
+    nvm install $NODE_VERSION && \
+    nvm alias default $NODE_VERSION && \
+    nvm use default
+
+ENV NODE_PATH "$NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules"
```
Comment (chrmod) [inline Dockerfile]: with nvm I don't think those envs are required

**Pull request merged**